common/scripts/configs/system_codes.sql

```diff
@@ -479,7 +479,7 @@ do $$
 
     if not exists (select 1 from "systemCodes" where code = 10001) then
       insert into "systemCodes" ("code", "name", "group", "createdBy", "updatedBy", "createdAt", "updatedAt")
-      values (10001, 'dynamic__x86_64__Ubuntu__14.04__c4.xlarge', 'clusterType', '54188262bc4d591ba438d62a', '54188262bc4d591ba438d62a', '2017-12-12', '2017-12-12');
+      values (10001, 'dynamic__x86_64__Ubuntu_14.04__c4.xlarge', 'clusterType', '54188262bc4d591ba438d62a', '54188262bc4d591ba438d62a', '2017-12-12', '2017-12-12');
```

Comment:
Extra underscore between Ubuntu and 14.04.
This has not made to a release yet, so I'll not remove the incorrect systemCode first.